.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,32 @@
+name: Bug report
+description: Create a new issue reporting a bug
```

Comment:
What do you think about adding the bug label here?

Reply:
It's a good idea! Added in fbaf003ea96a97d534ed5084e197825d5539df21.